Day-I-Social-Card/social-card/src/components/HeaderComponents/Header.css

```diff
@@ -1 +1,33 @@
 /* All styles for your header Components live here */
+.headerContainer {
```

Comment:
I know this is how the project was structured, but don't for a second think this is the only way to do things. There are better organization patterns you can find if you're brave enough.